wp-seo-main.php

```diff
@@ -332,7 +336,25 @@ function load_yoast_notifications() {
 function yoast_wpseo_self_deactivate() {
 	if ( is_admin() ) {
 		$message = esc_html__( 'The Standard PHP Library (SPL) extension seem to be unavailable. Please ask your web host to enable it.', 'wordpress-seo' );
-		add_action( 'admin_notices', create_function( $message, 'echo \'<div class="error"><p>\' . __( \'Activation failed:\', \'wordpress-seo\' ) . \' \' . $message . \'</p></div>\';' ) );
+		add_action( 'admin_notices', create_function( '', 'echo \'<div class="error"><p>\' . __( \'Activation failed:\', \'wordpress-seo\' ) . \' ' . $message . '</p></div>\';' ) );
+
+		deactivate_plugins( plugin_basename( WPSEO_FILE ) );
+		if ( isset( $_GET['activate'] ) ) {
+			unset( $_GET['activate'] );
+		}
+	}
+}
+
+/**
+ * Throw an error if the Composer autoload is missing and self-deactivate plugin
+ *
+ * @return void
+ */
+function yoast_wpseo_missing_autoload() {
+	if ( is_admin() ) {
+		$message = esc_html__( 'The WordPress SEO plugin installation is incomplete. Please refer to %1$sinstallation instructions%2$s.', 'wordpress-seo' );
```

Comment:
@Rarst please add a [translator comment](https://codex.wordpress.org/I18n_for_WordPress_Developers#Descriptions), like in https://github.com/Yoast/wordpress-seo/blob/jdevalk/issues/improve-a11y/admin/pages/metas.php#L187